### DISCUSSION TITLE
Can't install the tweak

### DISCUSSION THREAD

**Discussion by valentinldb** [SoCuul/SCInsta#121] [Support] [answered]
Hello, when I try to install SCInsta on AltStore it shows this message.

I'm on IOS 17.1 and iPhone 16 pro. Does anyone know how to solve this problem? 

`Error Domain=IXErrorDomain Code=2 "Failed to set app extension placeholders for com.burbn.instagram.C7AT65VR9G" UserInfo={NSLocalizedFailureReason=Failed to create promise., NSLocalizedDescription=Failed to set app extension placeholders for com.burbn.instagram.C7AT65VR9G, FunctionName=+[IXPlaceholder _placeholderForBundle:client:withParent:installType:metadata:placeholderType:mayBeDeltaPackage:isFromSerializedPlaceholder:error:], SourceFileLine=968, NSUnderlyingError=0xc822907e0 {Error Domain=IXErrorDomain Code=8 "Attempted to set app extension placeholder promise with bundle ID com.burbn.instagram.lockscreencamera that does not match required prefix of com.burbn.instagram.C7AT65VR9G. for parent" UserInfo={NSLocalizedDescription=Attempted to set app extension placeholder promise with bundle ID com.burbn.instagram.lockscreencamera that does not match required prefix of com.burbn.instagram.C7AT65VR9G. for parent, FunctionName=-[IXPlaceholder setAppExtensionPlaceholderPromises:error:], SourceFileLine=1582, NSLocalizedFailureReason=Mismatched bundle IDs.}}}`

**Comment by SoCuul** [comment]
Check out what's mentioned here: https://github.com/SoCuul/SCInsta/discussions/98

**Comment by valentinldb** [comment, marked as answer]
ah yes I hadn't seen thank you very much, for those who have the same problem as me, version 334.0 works perfectly (for me the 350.0 does not work)
